packages/codemirror-blocks/src/edits/fakeAstEdits.ts

```diff
@@ -170,90 +166,80 @@ export class FakeAstReplacement {
 
 const fakeNodeSpec = nodeSpec([value<string, "text">("text")]);
 // A fake ASTNode that just prints itself with the given text.
-class FakeInsertNode extends ASTNode<typeof fakeNodeSpec> {
```

Comment:
And this is all doable now that you've made it possible for arbitrary objects that meet a certain spec to be passed directly to ASTNode. Damn. That's _pretty_.